course_catalog/etl/prolearn_test.py

```diff
@@ -110,16 +110,19 @@ def test_prolearn_transform_programs(mock_csail_programs_data):
             "offered_by": [{"name": OfferedBy.csail.value}],
             "runs": [
                 {
-                    "run_id": program["nid"],
+                    "run_id": f"{program['nid']}_{start_val}",
                     "platform": PlatformType.csail.value,
                     "title": program["title"],
                     "offered_by": [{"name": OfferedBy.csail.value}],
                     "prices": parse_price(program),
-                    "start_date": parse_date(program["start_value"]),
-                    "end_date": parse_date(program["end_value"]),
-                    "best_start_date": parse_date(program["start_value"]),
-                    "best_end_date": parse_date(program["end_value"]),
+                    "start_date": parse_date(start_val),
+                    "end_date": parse_date(end_val),
+                    "best_start_date": parse_date(start_val),
+                    "best_end_date": parse_date(end_val),
```

Comment:
**Question:** If an offering has multiple runs, `run1, run2, run3,...` should
```
run1. best_start_date
run2. best_start_date
run3. best_start_date
```
all be the same? E.g., all point to `run2` if that is the best run?

Reply:
best_start_date  is specific to each run, and was meant to handle cases from other sources where there were multiple start dates, some of which might be blank, for example:
```
course_run.best_start_date = (
    course_run.enrollment_start
    or course_run.start_date
    or semester_year_to_date(course_run.semester, course_run.year)
)
```
But for prolearn there’s only 1 start date, so that’s what best_start_date should be too.

Reply:
Matt clarified that this is correct. Best start date isn't "the best run's start date", it's the best start date when there are multiple things that might be considered a start date:
```
course_run.best_start_date = (
    course_run.enrollment_start
    or course_run.start_date
    or semester_year_to_date(course_run.semester, course_run.year)
)
```

_I was thinking of the `findBestRun` function on frontend._